### PR TITLE
#166457882 Removes password field

### PR DESCRIPTION
**Pull request by Lundii**
#### What does this PR do?
It deletes the password field sent to the client when a user logs in
#### Description of Task to be completed?
* modified the AuthContoller file
#### How should this be manually tested?
* login as a registered user
* the password field is not returned as part of the returned object
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
[#166457882](https://www.pivotaltracker.com/story/show/166457882)
#### Screenshots (if appropriate)

image

#### Questions:
No

### CODE REVIEW

**Review by henryade — COMMENTED** [review]
Correct the PR name

File: server/controllers/AuthController.js [hunk 1]
```diff
@@ -26,13 +26,12 @@ class AuthController {
...
-      const { id } = user;
+      const { id, dataValues } = user;
       const token = createToken({ id });
-      delete user.password;
+      delete dataValues.password;
       return responseMessage(response, 200, {
         status: 'success',
-        user,
-        token
+        user: { ...dataValues, token }
```
Comment (KvNGCzA) [inline server/controllers/AuthController.js]: take token out of the user object

**Review by KvNGCzA — CHANGES_REQUESTED** [review]

**Comment by Lundii** [comment]
> Correct the PR name

Done

**Comment by henryade** [comment]
> > Correct the PR name
> 
> Done

Should the PR be named removes password field?

**Comment by Lundii** [comment]
> > > Correct the PR name
> > 
> > 
> > Done
> 
> Should the PR be named removes password field?

Please suggest one for me

> > > Correct the PR name
> > 
> > 
> > Done
> 
> Should the PR be named removes password field?

Corrected

**Review by KvNGCzA — APPROVED** [review]

**Pull request merged**